identity/client/src/pages/users/detail/authorization/List.tsx

```diff
@@ -47,23 +138,115 @@ const List: FC<AuthorizationsListProps> = ({ user, loadingUser }) => {
+                      handleOpenPatchModal({
+                        ownerKey: user.key,
+                        resourceType: null,
+                        permissionType: null,
+                        currentAuthorizations: rows,
+                      })
+                    }
+                  >
+                    {t("Add Authorizations")}
+                  </Button>
+                </TableToolbarContent>
+              </TableToolbar>
+              <Table {...getTableProps()}>
+                <TableHead>
+                  <TableRow>
+                    <TableExpandHeader />
+                    {headers.map((header) => (
+                      <TableHeader {...getHeaderProps({ header })}>
+                        {header.header}
+                      </TableHeader>
+                    ))}
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {rows.map((row) => {
+                    return (
+                      <React.Fragment key={row.id}>
+                        <TableExpandRow
+                          {...getRowProps({ row })}
+                          onClick={() => handleExpand(row.id)}
+                          isExpanded={expandedRows[row.id]}
+                        >
+                          <TableCell>
+                            {
+                              row.cells.find(
+                                (cell) => cell.info.header === "resourceType",
+                              )?.value
+                            }
+                          </TableCell>
+                        </TableExpandRow>
+                        {expandedRows[row.id] && (
+                          <TableExpandedRow colSpan={headers.length + 2}>
+                            {/* Displaying list of permissions */}
+                            {dataMap[row.id]?.permissions && (
+                              <ul>
+                                {dataMap[row.id].permissions.map(
+                                  (permission, index) => (
+                                    <React.Fragment key={index}>
+                                      <UnorderedList>
+                                        <StyledListItem>
+                                          <div>
+                                            <strong>Permission Type:</strong>{" "}
+                                            {permission.permissionType}
+                                            <br />
+                                            <strong>Resource IDs:</strong>{" "}
+                                            {permission.resourceIds.join(", ")}
+                                          </div>
+                                          <Button
+                                            kind="ghost"
+                                            size="small"
+                                            iconDescription={t(
+                                              "Edit permission",
```

Comment:
Don't we have to create translations keys for these strings?

Reply:
I checked the codebase and it seems there are a lot of uses of that translate function but no keys were ever set. I also think they have to be manually defined, from what I could find there's only a handful of keys defined in some json files in this folder `src/utility/localization/en`.

I believe this is not worth the effort right now, so I vote we leave this as is for now until we can discuss if we need translation and when do we need it by. Even if it was worth to do it, it should be a separate PR fixing all instances of this. What do you think @vsgoulart?